doc/concepts.rst

```diff
@@ -193,7 +193,8 @@ In addition, ``(action ...)`` fields support the following special variables:
 - ``exe:<path>`` is the same as ``<path>``, except when cross-compiling, in
   which case it will expand to ``<path>`` from the host build context
```

Comment:
See, this seems confusing. Shouldn't `run` use `exe:<path>` instead of `bin:<path>`? I'm assuming "host build context" has the "obvious meaning" of "context of non-cross-compiled programs", but since https://dune.readthedocs.io/en/stable/cross-compilation.html doesn't say otherwise (or mention _host_ build contexts), I guess that's right?